client/app/queue/PowerOfAttorneyDetail.jsx

```diff
@@ -120,16 +120,20 @@ export const PowerOfAttorneyDetailUnconnected = ({ powerOfAttorney, appealId, po
     });
   }
 
+  const show_poa_details = poa.representative_type && poa.representative_name
```

Comment:
frontend variables are usually camel-cased
```suggestion
  const showPoaDetails = poa.representative_type && poa.representative_name
```